src/sentry/incidents/subscription_processor.py

```diff
@@ -422,20 +422,24 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
                 data_packet = DataPacket[MetricDetectorUpdate](
                     source_id=str(self.subscription.id), packet=packet
                 )
-                results = process_data_packets([data_packet], DATA_SOURCE_SNUBA_QUERY_SUBSCRIPTION)
-                if features.has(
-                    "organizations:workflow-engine-metric-alert-dual-processing-logs",
-                    self.alert_rule.organization,
-                ):
-                    logger.info(
-                        "dual processing results for alert rule",
-                        extra={
-                            "results": results,
-                            "num_results": len(results),
-                            "value": aggregation_value,
-                            "rule_id": self.alert_rule.id,
-                        },
+                # temporarily skip processing any anomaly detection alerts
+                if self.alert_rule.detection_type != AlertRuleDetectionType.DYNAMIC:
+                    results = process_data_packets(
+                        [data_packet], DATA_SOURCE_SNUBA_QUERY_SUBSCRIPTION
                     )
+                    if features.has(
+                        "organizations:workflow-engine-metric-alert-dual-processing-logs",
+                        self.alert_rule.organization,
+                    ):
+                        logger.info(
+                            "dual processing results for alert rule",
+                            extra={
+                                "results": results,
+                                "num_results": len(results),
+                                "value": aggregation_value,
+                                "rule_id": self.alert_rule.id,
+                            },
+                        )
```

Comment:
Can you add a test like https://github.com/getsentry/sentry/blob/master/tests/sentry/incidents/test_subscription_processor.py#L3921-L3940 but with an anomaly detection alert?